dialdbg/README.md

```diff
@@ -0,0 +1,47 @@
+# Viam Dial Debugger
+A CLI tool to give information on how Viam rust utils' dial function makes connections.
+
+## Installation
+TODO(RSDK-3446): ensure installation constructions are correct.
+
+viam-dialdbg can be installed via homebrew.
+
+``` shell
+brew install viam-dialdbg
+```
+
+## Usage examples
+
+``` shell
+viam-dialdbg --uri myremote.com --credential mycredential
+```
+Prints debug information to STDOUT for connecting from this machine to "myremote.com" using "mycredential" as a credential. Prints information on WebRTC connection establishment, gRPC connection establishment, and average round-trip-times for both. Prints discovered mDNS addresses on the subnet if mDNS could not be used to connect.
+
+``` shell
+viam-dialdbg --uri myremote.com --credential mycredential --output ./foo.txt
+```
+Same as above, but outputs debug information to ./foo.txt (./foo.txt will be overwritten).
+
+``` shell
+viam-dialdbg --uri myremote.com --credential mycredential --output ./foo.txt --credential-type bar
+```
+Same as above, but uses "bar" credential type for "mycredential".
+
+``` shell
+viam-dialdbg --uri myremote.com --credential mycredential --output ./foo.txt --credential-type bar --nogrpc
+```
+Same as above, but only examines WebRTC connection establishment.
+
+Use `viam-dialdbg --help` for more information.
+
+## Prerequisites for development
+
+### Installing Rust for Mac and Unix-Like
```

Comment:
We have these instructions at the top level `README` already, so it may be redundant/slightly confusing to have them here as well.

Reply:
Removed.